### PR TITLE
Read me update

### PR DESCRIPTION
**Pull request by irishsheila**
Updated readme for iOS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,14 +19,21 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 [iOS Specific](https://github.com/realeyes-media/scte35-swift)
```
Comment (realeyes-mike-patterson) [inline README.md]: I think this link to the iOS equivalent project can be removed. And the "Android Specific" and "Javascript Specific" links are to test harnesses as well so the "iOS Test Harness" can be moved under the same heading as these

File: README.md [hunk 2]
```diff
@@ -19,14 +19,21 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
...
 
-[Javascript Specific](https://github.com/realeyes-media/scte35-js)
+
 
 ### How to Include on Your Platform
 
 
-**Javascript**
+
+## Javascript
```
Comment (realeyes-mike-patterson) [inline README.md]: Maybe 4 "#"s? This is a subheading under "How to Include on Your Platform" so it's weird that its heading font is bigger than that one

File: README.md [hunk 3]
```diff
@@ -57,11 +64,20 @@ console.log('from Base64 String: ', scteMessageFromBase64String(B64_STRING));
...
+
+To create & install a framework to use in iOS/Xcode:
+1. From the terminal, `cd` into the scte35-decoder-multiplatform project & type `./gradlew build` to run the the build command to build the framework. 
+2. Once the build has completed, the iOS framework can be found here: `scte35-decoder-multiplatform/build/cocoapods/framework/scte35decoder.framework`. 
+3. Inside a new or existing iOS Project in Xcode, from the Project Navigator (the leftmost panel in Xcode), select the project file. 
+4. In the center panel, under **TARGETS**, select the target in which the framework will be used. 
+5. Scroll down to **Frameworks, Libraries, and Embedded Content**, select the + button & navigate to `scte35decoder.framework` to add framework or drag file from Finder directly to this area.
+6. In each file where the framework will be used, be sure to add `import scte35decoder` to top of the file as part of the import statements.
+
+_Note: If Xcode has difficulties connecting to the framework, delete the framework from the Frameworks folder in Project Navigator, choosing "Remove Reference." Then physically drag & drop the framework directly into that folder & choose "copy if needed."_
```
Comment (realeyes-mike-patterson) [inline README.md:70]: Nice note

File: README.md [hunk 4]
```diff
@@ -57,11 +64,20 @@ console.log('from Base64 String: ', scteMessageFromBase64String(B64_STRING));
...
 ```
 
-**Android**
 
-**iOS**
 
-After running the build command, the iOS framework can be found here: `build/cocoapods/framework/scte35decoder.framework`. _When adding the framework to the project, be sure to choose "copy if needed."_
+## iOS
+
+
```
Comment (realeyes-mike-patterson) [inline README.md:61]: extra space?

File: README.md [hunk 5]
```diff
@@ -57,11 +64,20 @@ console.log('from Base64 String: ', scteMessageFromBase64String(B64_STRING));
...
+To create & install a framework to use in iOS/Xcode:
```
Comment (realeyes-mike-patterson) [inline README.md:62]: Awesome steps. Well-written and easy to understand. Feels like even a novice to iOS could follow these.

File: README.md [hunk 6]
```diff
@@ -110,7 +131,7 @@ class SCTEDecoder {
 	
 - You add the **GPR_USER** and **GPR_API_KEY** values to your environment variables on you local machine or build server to avoid creating a github properties file
 
-### Step 3: Configure Github pacakges
+### Step 3: Configure Github packages
```
Comment (realeyes-mike-patterson) [inline README.md:119]: Nice

File: scte35decoder.podspec [hunk 7]
```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'scte35decoder'
-    spec.version                  = '0.0.2'
-    spec.homepage                 = 'Link to a Kotlin/Native module homepage'
+    spec.version                  = '0.0.1'
```
Comment (realeyes-mike-patterson) [inline scte35decoder.podspec]: Why are we going back a version?

File: README.md [hunk 8]
```diff
@@ -92,7 +108,12 @@ class SCTEDecoder {
 }
 ```
 
-## Usage
+
+
+## Android
+
+
+### Usage
```
Comment (realeyes-mike-patterson) [inline README.md]: usage can probably just be deleted now

File: scte35decoder.podspec (hunk 7 above)
Comment (realeyes-mike-patterson) [inline scte35decoder.podspec]: why are we going back a version?

File: scte35decoder.podspec [hunk 9]
```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |spec|
     spec.libraries                = "c++"
     spec.module_name              = "#{spec.name}_umbrella"
 
-    spec.ios.deployment_target = '0.0.2'
+    spec.ios.deployment_target = '13.5'
```
Comment (realeyes-mike-patterson) [inline scte35decoder.podspec:15]: Nice

File: README.md (hunk 6 above)
Comment (realeyes-mike-patterson) [inline README.md:119]: Nice

File: README.md (hunk 5 above)
Comment (realeyes-mike-patterson) [inline README.md:62]: Steps seem really clearly written and in plain English. Nicely done

File: README.md (hunk 3 above)
Comment (realeyes-mike-patterson) [inline README.md:70]: Nice note.

File: README.md (hunk 1 above)
Comment (realeyes-mike-patterson) [inline README.md]: This one can be removed

File: README.md [hunk 10]
```diff
@@ -19,14 +19,21 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 [iOS Specific](https://github.com/realeyes-media/scte35-swift)
 
+[Javascript Specific](https://github.com/realeyes-media/scte35-js)
+
+
+
+### Test Harnesses
+
 [iOS Test Harness](https://github.com/realeyes-media/scte35-decoder-multiplatform-iOS-harness)
```
Comment (realeyes-mike-patterson) [inline README.md]: The other "Android specific" and "JS specific" links are to test harnesses so this link can actually be grouped with them under the "Reference Projects" section

File: README.md (hunk 1 above)
Comment (realeyes-mike-patterson) [reply README.md]: We don't link to the android kotlin github repo either fwiw

File: README.md (hunk 2 above)
Comment (realeyes-mike-patterson) [inline README.md]: It's odd that the subheadings under "How to Include on Your Platform" are given a bigger Heading status. Using 4 "#"s might be more appropriate.

**Review by realeyes-mike-patterson — COMMENTED** [review]
Looking really good. A couple minor changes could be made if you agree with the comments.

File: scte35decoder.podspec (hunk 7 above)
Comment (irishsheila) [reply scte35decoder.podspec]: Yeh, I don't know why even updated.

**Review by realeyes-mike-patterson — APPROVED** [review]
Looks good, thanks Sheila! Includes some extra fixes that are appreciated and the steps are clear and well-written
